cli/cmd/config.go

```diff
@@ -140,3 +144,27 @@ func getAPIClient() *openapi.APIClient {
 	}
 	return openapi.NewAPIClient(config)
 }
+
+func getResourceClient(resourceType string) (http.Client, http.Request) {
```

Comment:
As we don't have the open api specs I added this basic HTTP client to push YAML files to the backend services

Reply:
Instead of returning a request, you can return a custom type with a reference to the client and the url or any other info needed. Then each consumer can build their own request.